source/operator.rst

```diff
@@ -6,832 +6,111 @@ The operator is configured via the spec section of the
-|                                |           | ry-crede | imagePullS |
-|                                |           | ntials`` | ecret <htt |
-|                                |           |          | ps://kuber |
-|                                |           |          | netes.io/d |
-|                                |           |          | ocs/concep |
-|                                |           |          | ts/configu |
-|                                |           |          | ration/sec |
-|                                |           |          | ret/#using |
-|                                |           |          | -imagepull |
-|                                |           |          | secrets>`_ |
-|                                |           |          | _          |
-|                                |           |          | for the    |
-|                                |           |          | Percona    |
-|                                |           |          | XtraDB     |
-|                                |           |          | Cluster    |
-|                                |           |          | docker     |
-|                                |           |          | image      |
-+--------------------------------+-----------+----------+------------+
-| labels                         | label     | ``rack:  | The        |
-|                                |           | rack-22` | `Kubernete |
-|                                |           | `        | s          |
-|                                |           |          | affinity   |
-|                                |           |          | labels <ht |
-|                                |           |          | tps://kube |
-|                                |           |          | rnetes.io/ |
-|                                |           |          | docs/conce |
-|                                |           |          | pts/config |
-|                                |           |          | uration/as |
-|                                |           |          | sign-pod-n |
-|                                |           |          | ode/>`__   |
-+--------------------------------+-----------+----------+------------+
+
+
+.. csv-table:: PXC Section
+  :header: "Key", "Value", "Example", "Description"
+  :widths: 35, 8,20,25
+
+  "size", "int", ``3``, The size of the Percona XtraDB cluster must be >= 3 for `High Availability <https://www.percona.com/doc/percona-xtradb-cluster/5.7/intro.html>`_
+  "allowUnsafeConfigurations", "string",``false``, "Prevents users from configuring a cluster with unsafe parameters such as starting the cluster with less than 3 nodes or starting the cluster without TLS/SSL certificates"
+  image, string, ``percona/percona-xtradb-cluster-operator:1.0.0-pxc``, The Docker image of the Percona cluster used.
+  readinessDelaySec, int, ``15``, The delay before a check if the application is ready to process traffic
```

Comment:
this option also adds delay before run check ;)

Reply:
Added text.